.github/workflows/_bazel.yml

```diff
@@ -62,12 +70,20 @@ jobs:
           sudo apt-get install -y --no-install-recommends clang-16 lld-16 libunwind-16 libc++abi1-16 libc++1-16 libc++-16-dev libclang-rt-16-dev llvm-16
           sed -i -e "s%llvm-symbolizer%/usr/lib/llvm-16/bin/llvm-symbolizer%" .bazelrc
       - name: Setup Windows
-        if: inputs.os_name == 'windows'
+        if: runner.os == 'Windows'
         # Set a custom output root directory to avoid long file name issues.
         run: |
           git config --global core.symlinks true
           git config --show-scope --show-origin core.symlinks
           [System.IO.File]::WriteAllLines((Join-Path -Path $env:USERPROFILE -ChildPath '.bazelrc'), 'startup --output_user_root=C:/tmp')
+      - name: Setup macOS
+        if: runner.os == 'macOS'
+        run: |
+          sudo xcode-select -s "/Applications/Xcode_15.1.app"
```

Comment:
Test uses `macos-15`, where only Xcode 16 is available, so this won't work as-is – let's just land #3078 first so that they have the same OS version and the xcode-select command won't be needed

Reply:
Thank you for approving! I think ci-macOS-release will also need `--config=macos-cross-x86_64` now.